### ISSUE TITLE
Long running ML/Query Job Cancellation

### ISSUE THREAD

**Issue by tonyyang-svail** [sql-machine-learning/sqlflow#159]
Need to provide a way to kill/stop a long-running query/ML job.

**Comment by typhoonzero** [comment]
Should we implement something like?
```bash
SHOW JOBS;
|---|----------------|
| 0  | TRAIN iris.train |

KILL 0;
```

**Comment by hungry1526** [comment]
Good idea, @typhoonzero , also it would be better to add Job status, Job scheduler, Job type, running time, etc...

Another discussion point, should we expect user to start long running jobs in SQLFlow as it evolves. I am sure in the AntFin's case, there will be some.

**Comment by wangkuiyi** [comment]
I don't feel this is an urgent feature request.

Also, the design of SQLFlow keeps the flexibility to delegate submitter
programs to decide how and where to submit jobs. Suppose that a SQLFlow
deployment is configured to generate submitters that submit jobs to
Kubernetes, the user should be able to watch her jobs using Kubernetes
dashboard.

**Comment by wangkuiyi** [comment]
I would like to see that after submitting a job in Jupyter, it prints a URL to the Kubernetes dashboard.

**Comment by hungry1526** [comment]
> I would like to see that after submitting a job in Jupyter, it prints a URL to the Kubernetes dashboard.

In a single node setup, should SQLFlow return something like a process id? (If it's a long running job)
